lib/servers/script_scanner/script_scanner.py

```diff
@@ -20,9 +20,9 @@
 from twisted.internet.defer import inlineCallbacks, DeferredList, returnValue
 from script_signals_server import ScriptSignalsServer
 try:
-    from config.scriptscanner_config import config
+    import config.scriptscanner_config as sc_config
 except:
```

Comment:
This is required because you cannot reload a class you need to reload a module and importing only the class disconnects it from the module. Now when reload() is called the actual file with experiments updates the available experiments